internal/sql/service.go

```diff
@@ -168,9 +165,10 @@ func (r *Result) Next() (sql.Row, error) {
 	if err != nil {
 		return nil, err
 	}
+	r.qr.Metadata()
```

Comment:
Fail to see the usage.

Reply:
Great catch! Thanks. Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/755/commits/b5a568c30b94dd9156023089931b62d505fd0b7a